src/index.js

```diff
@@ -109,6 +109,10 @@ export default class Sound extends React.Component {
       if (this.props.volume !== prevProps.volume) {
         sound.setVolume(this.props.volume);
       }
+
+      sound.options.onfinish = () => this.props.onFinishedPlaying() || null;
+      sound.options.whileplaying = () => this.props.onPlaying(this) || null;
+      sound.options.whileloading = () => this.props.onLoading(this) || null;
```

Comment:
why not just
sound.options.onfinish = this.props.onFinishedPlaying
is "this" really needed in arguments?

Reply:
I saw the code.
It will be wrong this.
Use sound instead of this.